cli/src/commands/edit.rs

```diff
@@ -21,10 +21,12 @@ use crate::cli_util::{CommandHelper, RevisionArg};
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Edit a commit in the working copy
+/// Loads the contents of the specified revision and sets it as the active changeset.
 ///
-/// Puts the contents of a commit in the working copy for editing. Any changes
-/// you make in the working copy will update (amend) the commit.
+/// Note, it is recommended to instead to use `jj new` and `jj amend`. Creating a new
```

Comment:
Insert "generally [recommended]"? I think `jj edit` is very appropriate for the `git stash pop` use case (resume work after some brief work elsewhere)

Reply:
Sure, will do